{"name": "test_invalid_credentials[edge]", "status": "passed", "attachments": [{"name": "stdout", "source": "81ff85c6-35df-4375-b5ac-23c23ca8323b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "initialize_driver", "value": "'edge'"}], "start": 1735503241816, "stop": 1735503242461, "uuid": "89d691cc-aa5b-4019-97ac-588840205b50", "historyId": "69a95e92c7a86dff72a2b2a4eb261f35", "testCaseId": "2e5ca5771c4f6e0c27efee7a6f54eebc", "fullName": "Tests.test_login.TestLogin#test_invalid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('initialize_driver')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "SOF-LAP-PRL002"}, {"name": "thread", "value": "39924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_login"}]}